app/assets/stylesheets/common/base/emoji.scss

```diff
@@ -14,7 +14,7 @@ body img.emoji {
   left: 50%;
   top: 50%;
   width: 445px;
-  height: 264px;
+  min-height: 264px;
```

Comment:
![screenshot from 2015-11-12 10 26 29](https://cloud.githubusercontent.com/assets/4335742/11109097/f566087c-8927-11e5-9ba8-d2b0553d8057.png)

Note how the arrows do not have the right background.